libraries/chain/include/eosio/chain/backing_store/db_combined.hpp

```diff
@@ -207,7 +207,10 @@ class rocksdb_whole_db_table_collector {
          all_secondary_indices_;
 };
 
-auto process_all = []() { return true; };
+// This used to be "auto process_all = []() { return true; };"
+// g++10 correctly reports a link error of multiple definitions
+// as db_combined.hpp is included by mutiple source files. 
```

Comment:
I don't see that this comment adds anything. I would say remove it.

Reply:
I agree, the comment is frivolous.

Reply:
Removed.